src/pages/ContentScripts/features/contributor_button/DataNotFound.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const DataNotFound = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        color: 'var(--color-fg-muted)',
+      }}
+    >
+      <h3>Data Not Found</h3>
+      <div style={{ width: '300px', margin: '2em' }}>
+        <p>Possible reasons are:</p>
+        <ul style={{ marginLeft: '1em' }}>
+          <li>This repository is too new, so its data has not been generated</li>
+          <li>This repository is not active enough, so its data are not generated</li>
+        </ul>
+      </div>
+    </div>
+  );
+};
+
+export default DataNotFound;
```

Comment:
Let's move the `src/pages/ContentScripts/features/repo-networks/DataNotFound.tsx` to `src/pages/ContentScripts/components` and use it from there. It is important to reuse code instead of duplicate them